docs/references/api/resource_embedding.rst

```diff
@@ -1189,6 +1199,69 @@ You can use this to get the columns of a join table in a many-to-many relationsh
      }
    ]
 
+One-to-many and Many-to-many relationships
+------------------------------------------
+
+For these relationships, the spread embedded resource columns will be aggregated into arrays.
+
 .. note::
+  This is disabled by default, since aggregates can cause performance issues.
+  It can be activated by setting :ref:`db-aggregates-enabled` to ``true``.
 
-  The spread operator ``...`` is borrowed from the Javascript `spread syntax <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax>`_.
+.. code-block:: bash
+
+   # curl -g "http://localhost:3000/directors?select=first_name,...films(film_titles:title,film_years:year)&first_name=like.Quentin*"
+
+  curl --get "http://localhost:3000/directors" \
+    -d "select=first_name,...films(film_titles:title,film_years:year)" \
+    -d "first_name=like.Quentin*"
+
+.. code-block:: json
+
+   [
+     {
+       "first_name": "Quentin",
+       "film_titles": [
+         "Pulp Fiction",
+         "Reservoir Dogs"
+       ],
+       "film_years": [
+         1994,
+         1992
+       ]
+     }
+   ]
+
+The order of the values inside the resulting array is unspecified.
+However, if more than one embedded column is selected, `it is safe to assume <https://www.postgresql.org/message-id/15950.1491843689%40sss.pgh.pa.us>`_ that all of them will return the values in the same unspecified order.
+From the previous example, we can say that "Pulp Fiction" was made in 1994 and "Reservoir Dogs" in 1992.
+
+It is expected to get duplicated or null values inside the aggregated array.
+So you can use the ``..`` operator to solve this.
+For example, a film could get nominated in the same year for different competitions.
+To get only the distinct years we would do:
+
+.. code-block:: bash
+
+   # curl "http://localhost:3000/films?select=title,..competitions(competition_years:year)"
+
+  curl --get "http://localhost:3000/films" \
+    -d "select=title,..competitions(competition_years:year)"
+
+.. code-block:: json
+
+  [
+    {
+      "title": "Pulp Fiction",
+      "competition_years": [
+        1994,
+        1995
+      ]
+    },
+    ".."
+  ]
```

Comment:
I understand the use case, but why force this behavior on the spread operator? Couldn't this work by using [Stripped Nulls](https://docs.postgrest.org/en/v12/references/api/resource_representation.html#stripped-nulls) on the regular spread `...` instead?

```
$ curl --get "http://localhost:3000/films" \
    -H "Accept: application/vnd.pgrst.array+json;nulls=stripped" \
    -d "select=title,..competitions(competition_years:year)"
```